src/main/cli/settings.c

```diff
@@ -479,8 +468,8 @@ static const char * const lookupTableOffOnAuto[] = {
     "OFF", "ON", "AUTO"
 };
 
-const char* const lookupTableInterpolatedSetpoint[] = {
-    "OFF", "ON", "AVERAGED_2", "AVERAGED_3", "AVERAGED_4"
+const char* const lookupTableFeedforwardAveraging[] = {
+    "NONE", "2_POINT", "3_POINT", "4_POINT"
```

Comment:
Should this be `OFF` instead of `NONE`?

Reply:
agreed, good point.
have made it consistent.

Reply:
> Were it not for those considerations, I think OFF is preferable, but given those considerations, I would prefer to use NONE at this point in time, and doubt it would cause any user confusion.

I can see the point in your reasoning, but to me long term maintainability (i.e. doing it the way that does not take into account considerations around the one-time ocurrence of a user updating from 4.2 to 4.3) wins out here. Once all 4.2 users have updated this point is moot, but the sub---optimal choice will stay with us. And there really is no 'at this point in time' aspect, as if we go for `NONE` now there is no way to change this later without breaking users configurations, undoing all of the potential benefits that we get right now. This is what is called 'technical debt', and should be avoided if possible.
Instead, we can work around the problem you are trying to avoid by being explicit about this in the release notes.
Also, if we were to keep 'NONE' here, `FEEDFORWARD_AVERAGING_OFF` in https://github.com/betaflight/betaflight/pull/10723/files#diff-5eff8f578853d737dc511b60540bf8a8d01b181a2ca90a6bb4c52212e328202dR125 should be renamed to `FEEDFORWARD_AVERAGING_NONE`.

Reply:
Thanks for explaining your reasoning. I'll revert back to `OFF`, happy to do that.  Will make sure that the tuning guide is clear on this point also.